stable/awsebscsiprovisioner/values.yaml

```
@@ -26,32 +26,31 @@ registrar:
       repository: "quay.io/k8scsi/csi-node-driver-registrar"
       tag: "v1.1.0"
```

```suggestion
      tag: "v1.2.0"
```

https://kubernetes-csi.github.io/docs/node-driver-registrar.html

We will get rid of some errors and have the nodes correct registered.

```
Mar 14 10:35:05 ip-10-0-128-49.us-west-2.compute.internal kubelet[22146]: E0314 10:35:05.750404   22146 goroutinemap.go:150] Operation for "/var/lib/kubelet/plugins/ebs.csi.aws.com/csi.sock" failed. No retries permitted until 2020-03-14 10:35:13.750374648 +0000 UTC m=+20.368597010 (durationBeforeRetry 8s). Error: "RegisterPlugin error -- failed to get plugin info using RPC GetInfo at socket /var/lib/kubelet/plugins/ebs.csi.aws.com/csi.sock, err: rpc error: code = Unimplemented desc = unknown service pluginregistration.Registration"
```